src/index.js

```diff
@@ -33,40 +33,48 @@ class PlayerLoaderPlugin {
   }
 
   apply(compiler) {
-    compiler.hooks.emit.tapAsync('PlayerLoaderPlugin', (compilation, callback) => {
```

Comment:
View this PR by checking the *Hide whitespace changes* checkbox to avoid whitespace changes.